content/applications/finance/payment_acquirers/paypal.rst

```diff
@@ -113,6 +90,48 @@ Your Paypal account is ready!
      `link <https://sandbox.paypal.com/cgi-bin/customerprofileweb?cmd=_profile-language-encoding>`_
      to configure the encoding format in a test environment.
 
+Settings in Odoo
+================
+
+.. seealso::
+   - :ref:`payment_acquirers/add_new`
+
+Credentials tab
```

Comment:
Please don't structure the page following the tabs in Odoo; it's terrible. Instead, the parent "Payment acquirers" page should rather be purged of this structure too.